lldb/packages/Python/lldbsuite/test/make/Makefile.rules

```diff
@@ -13,6 +13,13 @@
 # the building of the a.out executable program.  For example,
 # DYLIB_ONLY := YES
 #
+# When specifying one of the DYLIB_*_SOURCES variables, DYLIB_NAME
+# controls the (platform-dependent) name of the produced dylib. E.g.,
+# on Darwin, if "DYLIB_NAME := foo", the generated dylib will be called
+# "foo.dylib".
```

Comment:
I think this is still missing the `lib` portion?

Reply:
whoops, forgot about that, thanks